File: Projects/Pr3Retrofit/app/src/main/java/com/nsw2022/pr3retrofit/RetrofitHelper.java
package com.nsw2022.pr3retrofit;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitHelper {

    public  static String baseUrl = "http://kobis.or.kr";

    public static Retrofit getRetrofitInstance(){
        Retrofit.Builder builder = new Retrofit.Builder();
        builder.baseUrl(baseUrl);
        builder.addConverterFactory(GsonConverterFactory.create());
        Retrofit retrofit = builder.build();

        return retrofit;
    }
}

File: Projects/TPQucikPlaceByKakaoSearchAPI/app/src/main/java/com/nsw2022/tpqucikplacebykakaosearchapi/network/RetrofitHelper.kt
package com.nsw2022.tpqucikplacebykakaosearchapi.network

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory
import retrofit2.create

class RetrofitHelper {
    companion object{
        fun getRetrofitInstance(baseUrl:String): Retrofit{
            val retrofit= Retrofit.Builder()
                .baseUrl(baseUrl)
                .addConverterFactory(GsonConverterFactory.create())
                .build()

            return retrofit
        }
    }
}